aiida/orm/implementation/general/calculation/job/__init__.py

```diff
@@ -295,6 +297,28 @@ def set_queue_name(self, val):
         else:
             self._set_attr('queue_name', six.text_type(val))
 
+    def set_account(self, val):
+        """
+        Set the account on the remote computer.
+
+        :param str val: the account name
+        """
+        if val is None:
```

Comment:
Why do we want to set `None` if in the `get` method, we will return `None` if the attribute does not exist. Seems like we don't need to create unnecessary attributes if it does not contain an actual value. Maybe that all the way down when attributes get stored that attributes with `None` will be skipped, but I would argue that it would be better to just do nothing here in the case of `None`. Same goes for `set_qos`. I think you just copied the implementation from `set_queue_name`, but I think we should just adapt that as well.

Reply:
This is true. I just copied and have no idea of the earlier design decisions. Happy to adapt whatever you decide to do.